packages/backend/src/ee/services/EmbedService/EmbedService.ts

```diff
@@ -316,13 +314,14 @@ export class EmbedService extends BaseService {
 
     async getDashboard(
         projectUuid: string,
-        embedToken: string,
+        account: Account<EmbedJwt>,
```

Comment:
can you explain how the controller knows the type is `Account<EmbedJwt>` and not `Account<Record<string,unknown>>`? I couldn't see the narrowing anywhere!